components/script/dom/trustedtypepolicyfactory.rs

```diff
@@ -85,6 +86,48 @@ impl TrustedTypePolicyFactory {
         // Step 9: Return policy.
         Ok(policy)
     }
+
+    /// <https://w3c.github.io/trusted-types/dist/spec/#abstract-opdef-get-trusted-type-data-for-attribute>
+    fn get_trusted_type_data_for_attribute(
+        element: QualName,
+        attribute: String,
+        attribute_namespace: Option<Namespace>,
+    ) -> Option<DOMString> {
+        // Step 1: Let data be null.
+        let mut data = None;
```

Comment:
I know that this isn't particularly idiomatic Rust, but this way we closely follow the spec text. Not sure what your preference is.